dallinger/prolific.py

```diff
@@ -505,7 +505,7 @@ def _req(self, method: str, endpoint: str, **kw) -> dict:
                 if endpoint == "/studies/":
                     # method="POST", endpoint="/studies/", json=payload
                     response = {
-                        "id": "study-id",
+                        "id": "66b0f8e34632badef5c8d1db",
```

Comment:
This is another small bug that only occurred at the very end when running a Dallinger demo in Prolific debug mode